src/vivarium_public_health/metrics/utilities.py

```diff
@@ -137,6 +137,12 @@ def get_age_bins(builder) -> pd.DataFrame:
 
     """
     age_bins = builder.data.load('population.age_bins')
+
+    # Works based on the fact that currently only models with age_start = 0 can include fertility
+    age_start = builder.configuration.population.age_start
+    min_bin_start = age_bins.age_group_start[np.asscalar(np.digitize(age_start, age_bins.age_group_end))]
+    age_bins = age_bins[age_bins.age_group_start >= min_bin_start]
+
```

Comment:
You also want to set the 'age_group_start' on the lower bin if necessary.  E.g. if we're using 5 year age bins in the data and the user specifies an age start of 32, the bin corresponding to the lowest age should be 32-35.

Reply:
Good point. Just added that + updated test to verify